app/components/view-section-table-row/macro.njk

```diff
@@ -23,9 +25,11 @@
     {% elif layout == 'vertical' %}
       <div {{ commonAttributes | safe }}>
         <div data-test-id="view-section-table-row-vertical">
-          <div data-test-id="view-section-table-row-title" class="nhsuk-u-font-weight-bold nhsuk-u-padding-bottom-2">
-            {{ params.questionTitle }}
-          </div>
+          {% if params.questionTitle %}
+            <div data-test-id="view-section-table-row-title" class="nhsuk-u-font-weight-bold nhsuk-u-padding-bottom-2">
+              {{ params.questionTitle }}
+            </div>
+          {% endif %}
```

Comment:
It would be nice to refactor this component so there is less duplication!

Reply:
Yeah, you're right